src/reducer/watchLaterReducerFunction.js

```diff
@@ -0,0 +1,25 @@
+export const watchLaterReducerFunction = (watchState, { type, payload }) => {
+  switch (type) {
+    case "ADD_TO_WATCH_LATER":
```

Comment:
Suggestions:
- You can create an action file and define all the variables there instead of giving string here "ADD_TO_WATCH_LATER". Otherwise you need to change in all the places if you are changing the name.
- Example
![image](https://user-images.githubusercontent.com/70130189/161194006-00a98668-5457-44b1-b0d2-c4ce185cc0b9.png)

Reply:
Okay